deploy/helm/magda-dev.yml

```diff
@@ -40,28 +40,11 @@ gateway:
     directives:
       scriptSrc:
       - "''self''"
-      - "''unsafe-inline''" # For VWO, hopefully we can remove this soon
```

Comment:
We now have the following CSP error in console:
```
Refused to execute inline script because it violates the following Content Security Policy directive: "script-src 'self' browser-update.org". Either the 'unsafe-inline' keyword, a hash ('sha256-lHjh6+8zaTcKepW6RLPXoOB5686KuJPwukiRVR+0zl4='), or a nonce ('nonce-...') is required to enable inline execution.

react-ga.js:893 Refused to load the script 'https://www.google-analytics.com/analytics.js' because it violates the following Content Security Policy directive: "script-src 'self' browser-update.org". Note that 'script-src-elem' was not explicitly set, so 'script-src' is used as a fallback.
```
So we don't have anything reply on `unsafe-inline` right?